common/OpTestSystem.py

```diff
@@ -1008,7 +1008,7 @@ def sys_ipmi_boot_system_to_petitboot(self, i_console):
 
         # Exiting to petitboot shell
         self.console.expect('Petitboot', timeout=BMC_CONST.PETITBOOT_TIMEOUT)
-        time.sleep(20)
+        self.console.expect('x=exit', timeout=10)
```

Comment:
@stewart-ibm We could instead wait for `Info: Connected to server!` which appears as soon as the client has a connection and (usually) means most devices are available too.